docs/host-and-deploy/index.md

```diff
@@ -0,0 +1,162 @@
+
+1. Locally, navigate to the [URL Rewrite Module downloads page](https://www.iis.net/downloads/microsoft/url-rewrite#additionalDownloads). For the English version, select **WebPI** to download the WebPI installer. For other languages, select the appropriate architecture for the server (x86/x64) to download the installer.
+1. Copy the installer to the server. Run the installer. Select the **Install** button and accept the license terms. A server restart isn't required after the install completes.
+
+**Configure the website**
+
+Set the website's **Physical path** to the Blazor app's folder. The folder contains:
+
+* The *web.config* file that IIS uses to configure the website, including the required redirect rules and file content types.
+* The app's static asset folder.
+
+**Troubleshooting**
+
+If a *500 Internal Server Error* is received and IIS Manager throws errors when attempting to access the website's configuration, confirm that the URL Rewrite Module is installed. When the module isn't installed, the *web.config* file can't be parsed by IIS. This prevents the IIS Manager from loading the website's configuration and the website from serving Blazor's static files.
+
+For more information on troubleshooting deployments to IIS, see [Troubleshoot ASP.NET Core on IIS](https://docs.microsoft.com/aspnet/core/host-and-deploy/iis/troubleshoot).
+
+### Nginx
+
+The following *nginx.conf* file is simplified to show how to configure Nginx to send the *Index.html* file whenever it cannot find a corresponding file on disk.
+
+```
+events { }
+http {
+    server {
+        listen 80;
+
+        location / {
+            root /usr/share/nginx/html;
+            try_files $uri $uri/ /Index.html =404;
+        }
+    }
+}
+```
+
+For more information on production Nginx web server configuration, see [Creating NGINX Plus and NGINX Configuration Files](https://docs.nginx.com/nginx/admin-guide/basic-functionality/managing-configuration-files/).
+
+### Nginx in Docker
+
+Add one line to the Dockerfile, as shown in the following example:
+
+```
+FROM nginx:alpine
+COPY ./bin/Release/netstandard2.0/publish /usr/share/nginx/html/
+COPY nginx.conf /etc/nginx/nginx.conf
+```
+
+### GitHub Pages
+
+To handle URL rewrites, add a *404.html* file with a script that handles redirecting the request to the *index.html* page. For an example implementation provided by the community, see [Single Page Apps for GitHub Pages](http://spa-github-pages.rafrex.com/) ([rafrex/spa-github-pages on GitHub](https://github.com/rafrex/spa-github-pages#readme)). An example using the community approach can be seen at [blazor-demo/blazor-demo.github.io on GitHub](https://github.com/blazor-demo/blazor-demo.github.io) ([live site](https://blazor-demo.github.io/)).
+
+There's also a potential issue when using a project site instead of an organization site. Project sites have a path, so the base tag in *index.html* should be updated from `/` to `/<repo-name>`.
```

Comment:
I believe it's a real issue not a "potential" issue. Suggested wording: "Also, when using a project site instead of an organization site, update the base tag in *index.html* from `/` to `/<repo-name>`."

Reply:
Actually, this issue isn't specific to GitHub pages. It also applies when hosted in an IIS vdir. So maybe we should have a short separate section that speaks general updating the base path similar to how we discuss the need for URL rewrite?

Reply:
I see what you mean in your earlier remark.